### ISSUE TITLE
Any interest to add rate limit support to this tap?

### ISSUE THREAD

**Issue by ricklamers** [Widen/tap-rest-api-msdk#21]
See title. Would you welcome a PR that includes basic support for them?

Nice job on the tap :)

**Comment by jlloyd-widen** [comment]
Definitely! I don't have any good api's to test that sort of thing on so if you have something in mind I'll definitely take a look at it.

**Comment by s7clarke10** [comment]
There has been some rate limiting introduced in this PR: https://github.com/Widen/tap-rest-api-msdk/pull/36. It may resolve you problem.
